### DISCUSSION TITLE
How do i uninstall

### DISCUSSION THREAD

**Discussion by iiFlickd** [baffler/Transparent-Twitch-Chat-Overlay#260] [Q&A]
I cant find anything to properly uninstall this

**Comment by baffler** [comment]
Search for Add or Remove programs in Windows (can also find it under Settings > Apps > Installed Apps). Search for "transparent" and it should find the app, click the 3 dots (...) on the right side and choose uninstall. 

If you can't find the app there, it could be in the Control Panel and search "uninstall" at the top-right. And choose "Uninstall a program". Search for "transparent" at the top-right and right-click the app and choose Uninstall.
